Widgets/WidgetViews.swift

```diff
@@ -216,31 +217,61 @@ struct SearchWidgetView: View {
     var body: some View {
         ZStack {
             Rectangle()
-                .fill(Color.widgetBackground)
+                .fill(Color(designSystemColor: .surface))
                 .accessibilityLabel(Text(UserText.searchDuckDuckGo))
 
             VStack(alignment: .center, spacing: 15) {
 
-                Image("WidgetDaxLogo")
+                Image("Logo")
                     .resizable()
                     .frame(width: 46, height: 46, alignment: .center)
                     .isHidden(false)
                     .accessibilityHidden(true)
 
                 ZStack(alignment: Alignment(horizontal: .trailing, vertical: .center)) {
 
-                    Capsule(style: .circular)
+                    RoundedRectangle(cornerSize: CGSize(width: 8, height: 8))
                         .fill(Color.widgetSearchFieldBackground)
-                        .frame(width: 123, height: 46)
+                        .frame(width: 126, height: 46)
 
-                    Image("WidgetSearchLoupe")
-                        .padding(.trailing)
+                    Image("Find-Search-20")
+                        .frame(width: 20, height: 20)
+                        .padding(.leading)
+                        .padding(.trailing, 13)
                         .isHidden(false)
-
+                        .accessibilityHidden(true)
+                        .foregroundColor(Color(designSystemColor: .textPrimary).opacity(0.5))
                 }
             }.accessibilityHidden(true)
         }
-        .widgetContainerBackground(color: .widgetBackground)
+        .widgetContainerBackground(color: Color(designSystemColor: .surface))
+    }
+}
+
+struct PasswordsWidgetView: View {
+    var entry: Provider.Entry
+
+    var body: some View {
+        ZStack {
+            Rectangle()
+                    .fill(Color(designSystemColor: .surface))
+                    .accessibilityLabel(Text(UserText.passwords))
+
+            VStack(alignment: .center, spacing: 14) {
+
+                Image("WidgetPasswordIllustration")
```

Comment:
```suggestion
                Image(.widgetPasswordIllustration)
```

Think you could use that here (and other places too if so inclined)